src/puml.ts

```diff
@@ -64,10 +64,10 @@ class PlantUMLTask {
     ]);
 
     this.task.stdout.on("data", (chunk) => {
-      let data = chunk.toString().trimRight(); // `trimRight()` here is necessary.
```

Comment:
Trimming here breaks the case where a read gives `"<text "` and the next `"fill=red>` or so. Trimming here gives `"<textfill=red>"`, which breaks breaks the resulting svg. It happened to me...